### PR TITLE
PC-1047: fix test issues

### PR DESCRIPTION
**Pull request by samyou-softwire**
﻿Add the ticket number below and uncomment
[Link to Jira ticket](https://beisdigital.atlassian.net/browse/PC-1047)

# Description

fix uncaught test issues
- I forgot to update the tests when changing the output for invalid adding users
- both tickets added a WithConsortia method, leading to a conflict

# Checklist

- [x] I have made any necessary updates to the documentation
- [x] I have checked there are no unnecessary IDE warnings in this PR
- [x] I have checked there are no unintentional line ending changes
- [x] I have added tests where applicable
- [x] If I have made any changes to the code, I have used the IDE auto-formatter on it
- [x] If I have made any changes to website flow, I have checked forward and back behaviour is still consistent

### CODE REVIEW

File: HerPortal.UnitTests/Builders/UserBuilder.cs [hunk 1]
```diff
@@ -53,5 +47,4 @@ public UserBuilder WithEmail(string email)
         user.EmailAddress = email;
         return this;
     }
-}
-
+}
```
Comment (jdgage) [inline HerPortal.UnitTests/Builders/UserBuilder.cs:50]: Is this auto-formatter removing this? A bit surprising

File: HerPortal.UnitTests/Builders/UserBuilder.cs [hunk 2]
```diff
@@ -29,12 +29,6 @@ public UserBuilder WithLocalAuthorities(List<LocalAuthority> localAuthorities)
         user.LocalAuthorities = localAuthorities;
         return this;
     }
-    
-    public UserBuilder WithConsortia(List<Consortium> consortia)
-    {
-        user.Consortia = consortia;
-        return this;
-    }
```
Comment (jdgage) [inline HerPortal.UnitTests/Builders/UserBuilder.cs:37]: Just not used? Should it be?

**Comment by jdgage** [comment]
As mentioned on Slack, could you update the test job's config so it runs on all PRs please?

File: HerPortal.UnitTests/Builders/UserBuilder.cs (hunk 1 above)
Comment (samyou-softwire) [reply HerPortal.UnitTests/Builders/UserBuilder.cs:50]: yea this is auto formatted

File: HerPortal.UnitTests/Builders/UserBuilder.cs (hunk 2 above)
Comment (samyou-softwire) [reply HerPortal.UnitTests/Builders/UserBuilder.cs:37]: the same function appears below; this is a duplicate

**Review by jdgage — CHANGES_REQUESTED** [review]
Warnings are shouting at us while we're here (multiple times - probably related to the other comment). Should we be using this?

File: .github/workflows/build-and-run-tests.yml [hunk 3]
```diff
@@ -1,16 +1,8 @@
...
-      - develop
-      - staging
-      - main
-  push:
-    branches:
-      - develop
-      - staging
-      - main
+- pull_request
+- push
```
[the remark this answers is not in the document]
Comment (samyou-softwire) [reply .github/workflows/build-and-run-tests.yml]: changing the push to be branch specific seems to have made it not verify the last commit, unless that's expected?

**Comment by samyou-softwire** [comment]
> Warnings are shouting at us while we're here (multiple times - probably related to the other comment). Should we be using this?
code warnings should now be fixed

**Review by jdgage — APPROVED** [review]
Think this looks good, if it's all working right (seems to be?). Will copy the action config changes over to the other repos at some point.

**Pull request merged**